barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.hpp

```diff
@@ -184,7 +188,7 @@ class ClientIVC {
 
     bool initialized = false; // Is the IVC accumulator initialized
 
-    ClientIVC(TraceSettings trace_settings = {});
+    ClientIVC(size_t num_circuits = 0, TraceSettings trace_settings = {});
```

Comment:
Can we avoid defaulting this? You can explicitly default initialize the member `num_circuits` to be safe